icu4c/source/test/intltest/loctest.cpp

```diff
@@ -3341,3 +3393,117 @@ void LocaleTest::TestBug11053UnderlineTimeZone() {
     Locale l8(locale_str.c_str());
     assertTrue((locale_str + " !l8.isBogus()").c_str(), !l8.isBogus());
 }
+
+void LocaleTest::TestUnd() {
+    IcuTestErrorCode status(*this, "TestUnd()");
+
+    static const char empty[] = "";
+    static const char root[] = "root";
+    static const char und[] = "und";
+
+    Locale empty_ctor(empty);
```

Comment:
We normally use camelCase variable names in ICU, but this is test code and I kind of like these...?